config/main.py

```diff
@@ -2930,6 +2931,56 @@ def del_vrf(ctx, vrf_name):
         config_db.set_entry('VRF', vrf_name, None)
 
 
+@vrf.command('add_vrf_vni_map')
+@click.argument('vrfname', metavar='<vrf-name>', required=True, type=str)
+@click.argument('vni', metavar='<vni>', required=True)
+@click.pass_context
+def add_vrf_vni_map(ctx, vrfname, vni):
```

Comment:
Could you have some test coverage for this case as well?

Reply:
ok will do.